File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        # Sort in place using [:] and sorted method use set method to remove duplicates
        nums[:] = sorted(set(nums))
        # Return the length
        return len(nums)

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int p = 0;
        // Iterate through list
        for(int i = 1 ; i < nums.length ; i++){
        // Check if number at original position is the same
            if(nums[p] != nums[i]){
                nums[p+1] = nums[i];
                p++;
            }
        }
        return p + 1;
    }
}